sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_operations/_patch.py

```diff
@@ -15,20 +17,20 @@ class TextTranslationClientOperationsMixin(TextTranslationClientOperationsMixinG
     @overload
     def translate(
```

Comment:
I played with the error being reported by pyright on this line and I'm not sure I agree with it. We do account for all the base overloads in this patch file and only add an additional body type which should be compatible (since we transform it to the model in the method, but possibly the type checker misses that). I think we can ignore the error aka add a `pyright: ignore[reportIncompatibleMethodOverride]` on the lines throwing the error in this file.

Reply:
Added and issue is resolved.